ouroboros-consensus-cardano/src/unstable-shelley-testlib/Test/Consensus/Shelley/MockCrypto.hs

```diff
@@ -29,23 +31,17 @@ import           Control.State.Transition.Extended (PredicateFailure)
 import           Ouroboros.Consensus.Ledger.SupportsProtocol
                      (LedgerSupportsProtocol)
 import qualified Ouroboros.Consensus.Protocol.Praos as Praos
+import           Ouroboros.Consensus.Protocol.Praos.AgentClient
+                     (AgentCrypto (..))
 import           Ouroboros.Consensus.Protocol.TPraos (TPraos)
 import           Ouroboros.Consensus.Shelley.Eras (ShelleyEra)
 import           Ouroboros.Consensus.Shelley.Ledger (ShelleyBlock,
                      ShelleyCompatible)
 import           Ouroboros.Consensus.Shelley.Protocol.Abstract (ProtoCrypto)
 import           Test.QuickCheck (Arbitrary)
 
--- | A mock replacement for 'StandardCrypto'
---
--- We run the tests with this mock crypto, as it is easier to generate and
--- debug things. The code is parametric in the crypto, so it shouldn't make
--- much of a difference. This also has the important advantage
--- that we can reuse the generators from cardano-ledger-specs.
-data MockCrypto
-
 instance Crypto MockCrypto where
-  type KES      MockCrypto = MockKES 10
+  type KES      MockCrypto = MockKES 128
```

Comment:
This was done in order to align with https://github.com/input-output-hk/kes-agent/blob/a212f98b9bc567ebf3c8472e62f28450990f7821/kes-agent/src/Cardano/KESAgent/Protocols/StandardCrypto.hs#L48

Reply:
This was reverted